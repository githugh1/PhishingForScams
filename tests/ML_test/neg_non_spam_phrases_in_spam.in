{
  "email_subject": "Invoice Payment",
  "email_body": "Your invoice payment is due...",
  "sender": "billing@email.com"
}
