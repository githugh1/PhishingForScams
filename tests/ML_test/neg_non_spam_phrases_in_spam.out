{
  "is_spam": false,
  "confidence": 0.15,
  "message": "This email is not considered spam."
}
